app/migrations/0001_initial.py

```diff
@@ -21,7 +21,6 @@ class Migration(migrations.Migration):
                 ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
                 ('email', models.EmailField(max_length=254, unique=True)),
                 ('name', models.CharField(max_length=50)),
-                ('is_dj', models.BooleanField(default=False)),
```

Comment:
normally illegal to change a past migration, but this is still initial phase so accepted